lib/index.js

```diff
@@ -11,5 +12,6 @@ var main = function (options, callback) {
 
 module.exports = main;
 module.exports.launch = main;
+module.exports.argv = argv;
```

Comment:
These are the respective changes in ember-cli-sauce: https://github.com/johanneswuerbach/ember-cli-sauce/compare/master...simplabs:update-saucie. This would require a major bump, yes.